### ISSUE TITLE
tslint.json file-name-casing rule shows incorrect warning when defining rules for multiple file types

### ISSUE THREAD

**Issue by jedster1111** [Microsoft/vscode-typescript-tslint-plugin#60]
![screen shot 2019-02-24 at 00 11 51](https://user-images.githubusercontent.com/25291974/53293278-d949f380-37c8-11e9-9940-342ced726e5f.jpg)

![screen shot 2019-02-24 at 00 11 55](https://user-images.githubusercontent.com/25291974/53293280-dd761100-37c8-11e9-9020-88a2ebf5391e.jpg)

![screen shot 2019-02-24 at 00 13 08](https://user-images.githubusercontent.com/25291974/53293296-0ac2bf00-37c9-11e9-8375-1b7178f1bfae.jpg)

When using the file-name-casing rule and passing an object as an argument, there is a warning stating:
```
Incorrect type. Expected "string".
Value is not accepted. Valid values: "camel-case", "pascal-case", "kebab-case".(1)
```

Passing in an object works and is documented in [the tslint docs here](https://palantir.github.io/tslint/rules/file-name-casing/).

[Simple repo here](https://github.com/jedster1111/tslint-file-name-casing-test)

_I'm not 100% sure where is the best place to report this so sorry if this isn't the right place._

**Comment by jedster1111** [comment]
I think I realised where I should have created this, so I've created a pull request now and will close this issue.
